packages/kit/src/runtime/server/page/index.js

```diff
@@ -192,6 +199,13 @@ export async function render_page(
 					throw load_error;
 				}
 			});
+
+			void promise.then((server_data) => {
+				data_serializer.serialize(i, server_data);
```

Comment:
I think if `data_serializer.serialize()` throws an error, it needs to be catched here and rethrown after line 241.